src/components/Chart/Editor/FormFields.js

```diff
@@ -38,125 +45,125 @@ export const FormFields = props => {
-                      value={value[property] || defaultProps[property] || ''}
-                      onChange={createOnDropdownChange}
-                      defaultProps={defaultProps}
-                      context={determineAxisContext(
-                        groupObject[property].parent,
-                        value,
-                        defaultProps
-                      )}
-                      parent={groupObject[property].parent}
-                      xNumberFormat={value.xNumberFormat}
-                      timeParse={value.timeParse}
-                    />
-                  )
-                }
-                if (groupObject[property].enum) {
-                  return (
-                    <CustomValueDropdown
-                      key={property}
-                      label={groupObject[property].title}
-                      items={groupObject[property].enum}
-                      value={value[property] || defaultProps[property] || ''}
-                      onChange={createOnDropdownChange(property)}
-                    />
-                  )
-                }
-                if (groupObject[property].type === 'array') {
-                  return (
-                    <TickField
-                      key={property}
-                      property={property}
-                      groupObject={groupObject}
-                      value={value[property] || defaultProps[property]}
-                      config={value}
-                      createOnFieldChange={createOnFieldChange}
-                      context={determineAxisContext(
-                        property,
-                        value,
-                        defaultProps
-                      )}
-                      timeParseDefault={defaultProps.timeParse}
-                    />
-                  )
-                }
-                if (groupObject[property].type === 'boolean') {
-                  return (
-                    <div key={property} style={{ marginTop: '20px' }}>
-                      <Checkbox
-                        checked={
-                          value[property] === undefined
-                            ? defaultProps[property]
-                            : value[property]
-                        }
-                        onChange={createOnFieldChange(property)}
-                      >
-                        {groupObject[property].title}
-                      </Checkbox>
-                    </div>
-                  )
-                }
-                if (groupObject[property].format === 'Slider') {
-                  return (
-                    <Slider
-                      key={property}
-                      label={
-                        groupObject[property].title +
-                        ' ' +
-                        (value[property] || defaultProps[property])
+          <div {...styles.box} key={group}>
+            <div {...styles.title}>{fields[group].title}</div>
```

Comment:
Let's not use divs for font styles, alt without new styles
```suggestion
            <Interaction.P><strong>{fields[group].title}</strong></Interaction.P>
```